app/Kconfig

```diff
@@ -217,6 +218,56 @@ endif
 #ZMK_SPLIT_BLE
 endif
 
+# ZMK_BLE
+endif
+
+if ZMK_USB
```

Comment:
I'm not sure this really needs to be behind this condition, TBH.

Reply:
Do you mean we can remove this check ?
I added this check for cases where we have controllers with chips without having USB peripheral like nrf52832 chip.
If we are making an assumption that we support only USB controllers, we can remove this.
Let me know if i mis-understood the question

Reply:
I want to use nice nano in USB mode without bluetooth